charts/opensearch-dashboards/Chart.yaml

```diff
@@ -15,13 +15,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.9.1
+version: 1.10.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.3.6"
+appVersion: "1.3.7"
```

Comment:
AFAIK we do not change the app version right? @peterzhuamazon

Reply:
App Version is the docker image version.
Change it.

Reply:
Since 1.3.7 is out to public right now, thanks.